firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/ToolConfig.kt

```diff
@@ -22,4 +22,4 @@ package com.google.firebase.vertexai.type
  *
  * @param functionCallingConfig The config for function calling
  */
-public class ToolConfig(internal val functionCallingConfig: FunctionCallingConfig)
+public class ToolConfig(internal val functionCallingConfig: FunctionCallingConfig?)
```

Comment:
Should this be public?

```suggestion
public class ToolConfig(public val functionCallingConfig: FunctionCallingConfig?)
```

Reply:
Nope.

Reply:
It seems ios [also does this](https://github.com/firebase/firebase-ios-sdk/blob/6059bbbc98a488c096d17c002fae181ab6a548e1/FirebaseVertexAI/Sources/FunctionCalling.swift#L125), so SGTM